### PR TITLE
Add clipping planes

### PR DESCRIPTION
**Pull request by PixelDust22**
app 4_19_2023 2_49_50 PM

### CODE REVIEW

File: examples/static.rs [hunk 1]
```diff
@@ -39,4 +41,19 @@ fn startup(mut commands: Commands, asset_server: Res<AssetServer>) {
             Vec3::new(3.0, 3.0, 3.0),
             Vec3::ZERO,
         ));
+
+    commands.spawn(ClippingPlaneBundle {
+        range: ClippingPlaneRange {
+            min_sdist: 0.0,
+            max_sdist: 0.5,
```
Comment (bonsairobo) [inline examples/static.rs:48]: Huh this must be a small model, the screenshot shows a large portion of the model.

File: src/pipeline.rs [hunk 2]
```diff
@@ -50,19 +53,28 @@ pub(crate) fn prepare_point_cloud_bind_group(
...
-            label: Some("point_cloud_bind_group"),
-            layout: &pipeline.view_layout,
-            entries: &[BindGroupEntry {
-                binding: 0,
-                resource,
-            }],
-        });
-        bind_groups.bind_group = Some(bind_group);
+    if let Some(view_uniform_resource) = view_uniform.uniforms.binding() {
+        if let Some(clipping_plane_resource) = clipping_planes_uniform.0.binding() {
```
Comment (bonsairobo) [inline src/pipeline.rs]: You could do `if let (Some(_), Some(_)) =`

**Review by bonsairobo — APPROVED** [review]
Nice!

File: examples/static.rs (hunk 1 above)
Comment (PixelDust22) [reply examples/static.rs:48]: Yeah the model is normalized to 0-1

**Pull request merged**